config.json.example

```diff
@@ -12,5 +12,31 @@
     "initial_transfer": 0,
     "location_cache": true,
     "distance_unit": "km",
-    "item_filter": ["101","102","103","104"]
+    "item_filter": ["101","102","103","104"],
+    "release_config": {
+        "any": {
+            "release_under_cp": 400,
+            "release_under_iv": 0.8,
+            "cp_iv_logic": "or"
+        },
+        "My Pokemon": {
```

Comment:
This is not clear enough. Always put the default values in. Adding all the pokemon names would be more clear